conf_parse.go

```diff
@@ -90,16 +90,17 @@ func confFromStr(conf string) (*lineSenderConfig, error) {
 		case "token_y":
 			// Some clients require public key.
 			// But since Go sender doesn't need it, we ignore the values.
+			continue
```

Comment:
Semi-related, without this, we could accidentally set `auto_flush` settings through `token_y` and `token_x` values (if someone sets the value to an int or "off")